pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -99,7 +100,18 @@ func (lp *ProxyLifecycleProcessor) OnInvokeEnd(_ *invocationlifecycle.Invocation
 }
 
 func (lp *ProxyLifecycleProcessor) spanModifier(lastReqId string, chunk *pb.TraceChunk, s *pb.Span) {
-	// Add appsec tags to the aws lambda function service entry span
+	// Add relevant standalone tags to the chunk (TODO: remove per span tagging once backend handles chunk tags)
+	if config.IsStandalone() {
```

Comment:
Even if you optimize `IsStandalone()` to make it efficient for this call, you could also store at init-time the global span tags in `lp` and use them here instead of calling this function.